react-scanner.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests and stories.
+	globs: [ '**/!(test|stories)/!(*stories).@(js|ts)?(x)' ],
```

Comment:
All the `.ts` files are failing to parse, and there's very little chance that there will be component usage in a `.ts` file.

```suggestion
	globs: [ '**/!(test|stories)/!(*stories).@(js|jsx|tsx)' ],
```

Reply:
Makes sense 👍